Makefile

```diff
@@ -13,16 +14,16 @@ fioctl-static:
 
 fioctl-linux-amd64:
 fioctl-linux-arm64:
-fioctl-linux-armv7:
+fioctl-linux-arm-7:
 fioctl-windows-amd64:
 fioctl-darwin-amd64:
 fioctl-darwin-arm64:
 fioctl-%:
-	CGO_ENABLED=0 \
-	GOOS=$(shell echo $* | cut -f1 -d\- ) \
-	GOARCH=$(shell echo $* | cut -f2 -d\-) \
-		go build $(LDFLAGS) -o bin/$@ main.go
-	@if [ "$@" = "fioctl-windows-amd64" ]; then mv bin/$@ bin/$@.exe; fi
+	@test -x $(builder) || (echo "Please install xgo toolchain $(HOME)/go/bin: go install github.com/crazy-max/xgo@v0.30.0")
+	$(eval GOOS:=$(shell echo $* | cut -f1 -d\- ))
+	$(eval GOARCH:=$(shell echo $* | cut -f2- -d\-))
+	$(builder) --targets=$(GOOS)/$(GOARCH) -out bin/fioctl $(LDFLAGS) .
+	# This creates files as root, use `sudo chown --reference bin bin/fioctl-*` if you wish them under your user.
```

Comment:
I don't suppose xgo has a `--user` type option? This will probably only be a problem for someone like myself who has their code on a NFS volume that blocks root access. It's not a blocker, but will change my workflow

Reply:
This actually breaks the CI... so I had to make those files owned by a "real" user.

A problem with crazy-max/xgo is that it does not allow customizing docker args (like the original fork).
I've tried several options, and the one which works the most reliably is to alias the `docker` to pass in a modified `--entrypoint` argument.  I'll push that local commit on Monday.
A plain `--user` thingy did not work properly, it fails to operate inside the container as it tries to create some root-level folders.

Reply:
That's painful...